[2024-12-18T22:10:34.539+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.env_preparation manual__2024-12-18T22:10:32.945085+00:00 [queued]>
[2024-12-18T22:10:34.549+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.env_preparation manual__2024-12-18T22:10:32.945085+00:00 [queued]>
[2024-12-18T22:10:34.550+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-12-18T22:10:34.568+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): env_preparation> on 2024-12-18 22:10:32.945085+00:00
[2024-12-18T22:10:34.575+0000] {standard_task_runner.py:60} INFO - Started process 1136 to run task
[2024-12-18T22:10:34.578+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'env_preparation', 'manual__2024-12-18T22:10:32.945085+00:00', '--job-id', '105', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline_dag.py', '--cfg-path', '/tmp/tmp87ye0w0o']
[2024-12-18T22:10:34.582+0000] {standard_task_runner.py:88} INFO - Job 105: Subtask env_preparation
[2024-12-18T22:10:34.636+0000] {task_command.py:423} INFO - Running <TaskInstance: data_pipeline.env_preparation manual__2024-12-18T22:10:32.945085+00:00 [running]> on host 1d3bd1c7da18
[2024-12-18T22:10:34.715+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='env_preparation' AIRFLOW_CTX_EXECUTION_DATE='2024-12-18T22:10:32.945085+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-18T22:10:32.945085+00:00'
[2024-12-18T22:10:34.727+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-12-18T22:10:34.737+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=env_preparation, execution_date=20241218T221032, start_date=20241218T221034, end_date=20241218T221034
[2024-12-18T22:10:34.792+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-12-18T22:10:34.823+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
